src/OSPSuite.Presentation/Binders/ObservedDataDragDropBinder.cs

```diff
@@ -24,6 +25,25 @@ public IReadOnlyList<DataRepository> DroppedObservedDataFrom(IDragEvent e)
          return getObservedDataNodesFrom(e).Select(observedDataNode => observedDataNode.Tag.Subject).ToList();
       }
 
+      public Cache<string, List<DataRepository>> DroppedObservedDataWithFolderPathFrom(IDragEvent e)
+      {
+         var dataNodes = e.Data<IEnumerable<ITreeNode>>();
+
+         if (dataNodes == null)
+            return new Cache<string, List<DataRepository>>();
+
+         var treeNodes = dataNodes as IList<ITreeNode> ?? dataNodes.ToList();
```

Comment:
Can you explain why you don't just do ToList?

Reply:
Tbh I have not changed this part of code (or even the naming underneath for that matter). We were already differentiating between only folders and only data sets, because different methods were being used to retrieve the data already, all that changed was the signature(Cache) and now the naming to better reflect wehat the methods are doing. 

But I totally agree, I also see no reason why not just a  .ToList() is not being used. If you have nothing against this, I will change this to .ToList() only.

Reply:
Actually let us better briefly talk about this